lib/chromedriver.js

```diff
@@ -4,42 +4,145 @@ import events from 'events';
 import { JWProxy } from 'appium-base-driver';
 import cp from 'child_process';
 import { system, fs, logger } from 'appium-support';
-import { retryInterval } from 'asyncbox';
-import { SubProcess } from 'teen_process';
+import { retryInterval, asyncmap } from 'asyncbox';
+import { SubProcess, exec } from 'teen_process';
 import B from 'bluebird';
-import { getChromedriverBinaryPath } from './install';
+import { getChromedriverBinaryPath, getChromedriverDir } from './install';
+import { getChromeVersion } from './utils';
+import semver from 'semver';
+import _ from 'lodash';
 
 
 const log = logger.getLogger('Chromedriver');
 
 const DEFAULT_HOST = '127.0.0.1';
 const DEFAULT_PORT = 9515;
+const CHROMEDRIVER_CHROME_MAPPING = {
```

Comment:
this doesn't really solve the problem of Appium not supporting versions of chrome which got released after the release of Appium. does it? Can we put that into a json which may get downloaded from git?

Reply:
No, and it's not intended to. That can be handled with `chromedriverExecutable`.